song_match/song_match.py

```diff
@@ -37,22 +39,73 @@ def __init__(self):
 
         self.__turn_on_cube_lights()
 
-        notes = [Note('E4'), Note('D4'), Note('C4')]
-        await self._robot.play_notes(notes)
+        # notes = [Note('E4'), Note('D4'), Note('C4')]
```

Comment:
Did you mean to keep these comments in here?

Reply:
No, I didn't. Planned to delete them but forgot.